.vscode/launch.json

```diff
@@ -8,8 +8,7 @@
       "request": "launch",
       "runtimeExecutable": "${execPath}",
       "args": [
-        "--extensionDevelopmentPath=${workspaceRoot}/dist/vscode-codeql",
-        "--disable-extensions"
```

Comment:
I don't quite understand the reason for removing `--disable-extensions`? I thought we had it in here to ensure that integration test failures couldn't be because some other extension was confusing things.

Reply:
Our extension now depends on the "VS Code Test Explorer" extension. If we launch VS Code with `--disable-extensions`, it disables the "VS Code Test Explorer" extension, and therefore refuses to load our own extension due to the disabled dependency.